apps/account-api/src/services/accounts.service.ts

```diff
@@ -41,7 +47,7 @@ export class AccountsService {
     }
   }
 
-  async getMsaIdForPublicKey(publicKey: string): Promise<MsaIdResponse | null> {
+  async getMsaIdForAccountId(publicKey: string): Promise<MsaIdResponse | null> {
```

Comment:
name change for consistency

Reply:
That’s a great idea! I think we could also update the name of the parameter, maybe to something like "account."